gef-ui/templates/index.njk

```diff
@@ -2,7 +2,13 @@
 <html lang="en" class="govuk-template ">
   <head>
     <meta charset="utf-8" />
-    <title {% if pageTitleLang %} lang="{{ pageTitleLang }}"{% endif %}>{% block pageTitle %}{% endblock %} - UK Export Finance</title>
+    <title
+      {% if pageTitleLang %}
+        lang="{{ pageTitleLang }}"
+      {% endif %}
+      >
```

Comment:
Indentation out, should match `<title` please.

Reply:
I fixed indentation, but after more research, we decided to move `<title>` back to single line.

I also removed `pageTitleLang` because we don't use it.